lib/mix/tasks/expo.msguniq.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Mix.Tasks.Expo.Msguniq do
+  @shortdoc "Unify duplicate translations in message catalog"
+
+  @moduledoc """
+  Unify duplicate translations in message catalog
+
+      mix expo.msguniq [PO_FILE]
+
```

Comment:
@whatyouhide Would it make sense to align with gettexts `msguniq` instead?

`-o` / `--output` define the output location. `-` is `STDOUT`, which is also the default.

Reply:
No, I don't think so. We don't really align with GNU Gettext CLI tools in general, and `--output` is a bit annoying to use because I think you'll mostly do `expo.msguniq PO_FILE --output PO_FILE`, right?

Reply:
While the elixir `gettext` library takes a more opinionated approach, `expo` is quite aligned to GNU gettext. (Or at least I tried to :sweat_smile: ) 

You can see that with [`mix expo.msgfmt`](https://github.com/elixir-gettext/expo/blob/main/lib/mix/tasks/expo.msgmft.ex#L11), where all supported options are exactly as defined in GNU gettext: https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html

The `msgfmt` task also uses `--output-file`.

I would prefer to stay consistent across all tasks over having to specify the file twice.